### ISSUE TITLE
Add several exclusion patterns

### ISSUE THREAD

**Issue by 35609902357** [einaregilsson/Redirector#227]
At the moment only one exclusion pattern can be provided, which is a problem when using wildcards.

For example if I want to redirect `https://www.reddit.com/r/*` to `https://old.reddit.com/r/$1`, but I don't want "all", "pics" and "videos" subreddits to be redirected, I can't do it, I can only input one url to exclude. Wildcards/regex don't help because these paterns aren't regular

It would be nice to have a "+" button to add another `Exclude pattern` field to input another url that can't be covered by wildcards/regex.

Or to have something like linebreaks:

    https://www.reddit.com/r/all
    https://www.reddit.com/r/pics
    https://www.reddit.com/r/videos

or comma separated values , like:

`https://www.reddit.com/r/all,https://www.reddit.com/r/pics, https://www.reddit.com/r/videos`

**Comment by einaregilsson** [comment]
Why not use a regex: ```https://www\.reddit\.com/r/(all|pics|videos)``` ?

**Comment by 35609902357** [comment]
I input that line as an exclusion pattern, but it doesn't work. Maybe because the redirect is using wildcard instead of regex?

**Comment by einaregilsson** [comment]
Yeah, you need to use regex patterns for this. Wildcard patterns are only to do the most simple stuff, anything more than that you need regex.

**Comment by 35609902357** [comment]
That's why several exclusion patterns are useful, you can combine the simplicity of wildcards, but can exclude urls as needed. Instead of rewriting every filter as regex, which are more complicated.

**Comment by einaregilsson** [comment]
Sorry, I don't want to add more complexity to the UI. For more complex things you'll have to use regex.

**Comment by 35609902357** [comment]
How does having the possibility of adding urls separated by comma instead of only one url complicate UI?

`https://exampleurl1.com`

vs

`https://exampleurl1.com,https://exampleurl2.com,https://exampleurl3.com`

**Comment by ANIMka** [comment]
What if I need to add several sites to exceptions?

**Comment by stdedos** [comment]
On a tangent, I'd like "multiple examples" (in the "test-driven development" sense)
